pkg/server/endpoints/bundle/internal/acmetest/ca.go

```diff
@@ -62,10 +62,22 @@ import (
 	"sync"
 	"time"
 
-	"github.com/go-jose/go-jose/v3"
+	"github.com/go-jose/go-jose/v4"
 	"golang.org/x/crypto/acme"
 )
 
+var allowedJWTSignatureAlgorithms = []jose.SignatureAlgorithm{
```

Comment:
I thought it would be safer to have dedicated lists of allowed algorithms for each type of JWT we have to verify since the various providers of those JWTs could use different signature algorithms